### PR TITLE
[type: BUG] fix fallback issue

### PR DESCRIPTION
**Pull request by RayayChung**
Make sure that:

- [ ] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-common/src/main/java/org/apache/shenyu/common/utils/UriUtils.java [hunk 1]
```diff
@@ -42,6 +42,18 @@ public static URI createUri(final String uri) {
...
 
+    /**
+     * create URI {@link URI}.
+     *
+     * @param scheme    scheme eg:http
+     * @param authority registry or server eg: 127.0.0.1:8080
+     * @param path      path eg:/ fallback
+     * @return created {@link URI} from uri
+     */
+    public static URI createUri(final String scheme, final String authority, final String path) {
```
Comment (moremind) [inline shenyu-common/src/main/java/org/apache/shenyu/common/utils/UriUtils.java:53]: you can use springframework `UriComponentBuilder`

**Review by moremind — CHANGES_REQUESTED** [review]

File: shenyu-common/src/main/java/org/apache/shenyu/common/utils/UriUtils.java (hunk 1 above)
Comment (RayayChung) [reply shenyu-common/src/main/java/org/apache/shenyu/common/utils/UriUtils.java:53]: ok, i'll fix it

Comment (RayayChung) [reply shenyu-common/src/main/java/org/apache/shenyu/common/utils/UriUtils.java:53]: As this class `UriUtils` is in `shenyu-common`, can i add `spring-web` dependency into `shenyu-common` module?

Comment (moremind) [reply shenyu-common/src/main/java/org/apache/shenyu/common/utils/UriUtils.java:53]: no,

Comment (RayayChung) [reply shenyu-common/src/main/java/org/apache/shenyu/common/utils/UriUtils.java:53]: ok, i'll not fix it now

Comment (RayayChung) [reply shenyu-common/src/main/java/org/apache/shenyu/common/utils/UriUtils.java:53]: @moremind  can we skip this change?

**Review by moremind — APPROVED** [review]

**Comment by moremind** [comment]
LGTM

**Pull request merged**